include/cpp-statsd-client/StatsdClient.hpp

```diff
@@ -103,6 +140,13 @@ std::string sanitizePrefix(std::string prefix) {
     }
     return prefix;
 }
+
+// All supported metric types
+constexpr char METRIC_TYPE_COUNT[] = "c";
+constexpr char METRIC_TYPE_GUAGE[] = "g";
+constexpr char METRIC_TYPE_TIMING[] = "ms";
+constexpr char METRIC_TYPE_HISTOGRAM[] = "h";
+constexpr char METRIC_TYPE_SET[] = "s";
```

Comment:
here we enumerate all of the possible metric types rather than hardcode them in the various methods i thougt it was nicer to have them all together hidden in the detail namespace